src/sections/blog-post.liquid

```diff
@@ -0,0 +1,155 @@
+{%- comment -%}
+  This is a required section for the Shopify Theme Store.
+  It is loaded when you add "Blog Posts" section on the Theme Editor.
+
+  We are showing a preview of what the blog post looks like in the preview.
+
+  Required settings
+  - Blog (which blog category to show in your store)
+  - Number of posts to show
+
+  Optional settings
+  - Show author
+  - Show date
+{%- endcomment -%}
+<section>
+  {% if section.settings.title != blank %}
+    <h2>{{ section.settings.title | escape }}</h2>
+  {% endif %}
+
+  {%- comment -%}
+    This refer from blog section when you add the section from Theme Editor
+  {%- endcomment -%}
+  {%- assign blog = blogs[section.settings.blog] -%}
+
+  {% unless blog.articles_count == 0 %}
+    <div class="blog-post">
+      {% for article in blog.articles limit: section.settings.post_limit %}
+        <div class="blog-post__item">
```

Comment:
We don't have any styles for this class, so may as well omit it?

Reply:
Good point.